linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands/oval/shared.xml

```diff
@@ -23,23 +23,19 @@
 
   <!-- First define OVAL entities that can be reused across tests below -->
   <unix:file_object id="object_system_privileged_commands" comment="system files with setuid or setgid permission set" version="1">
-    <unix:behaviors recurse="directories" recurse_direction="down" recurse_file_system="local" max_depth="-1" />
-    <unix:path operation="equals">/</unix:path>
+    <unix:behaviors recurse_file_system="local"/>
+    <!-- don't search /dev, /proc, /sys -->
+    <unix:path operation="pattern match">^\/(?!(dev|proc|sys)(\/|$)).*$</unix:path>
```

Comment:
I think the forward slash need to be escaped, we don't escape it in this rule, for example.
https://github.com/ComplianceAsCode/content/blob/da57b9e50cfd6630116f169355aa14a4f5f54adf/linux_os/guide/system/software/integrity/software-integrity/aide/aide_scan_notification/oval/shared.xml#L43

And the second group doesn't need to capture anything. (Not sure if this would speed up the regex matching)

```suggestion
    <unix:path operation="pattern match">^/(?!(dev|proc|sys)(?:/|$)).*$</unix:path>
```

Reply:
I would like to have this as an OVAL macro, so it could be reused with less regex copy-pasting, and the intention would be totally obvious.

Reply:
Err, well, after https://github.com/OpenSCAP/openscap/pull/1726 we can throw away `/sys` and `/proc`. And this approach is in general 10-20% slower (most likely because of the regexp handling during traversal).